scripts/metavar.py

```diff
@@ -289,7 +289,11 @@ def valid_value(self, test_value, error=False):
                 pass
         elif self.type is bool:
             if isinstance(test_value, str):
-                valid_val = (test_value in ['True', 'False']) or (test_value.lower() in ['t', 'f', '.true.', '.false.'])
+                if test_value.lower() in ['t', 'true', '.true.'] + ['f', 'false', '.false.']:
+                    valid_val = (test_value.lower() in ['t', 'true', '.true.']) or \
+                                not (test_value.lower() in ['f', 'false', '.false.'])
```

Comment:
This could be more efficient if there was a variable for `__true_vals` and one for `__false_vals`.

Reply:
I will add these variables, thanks.